src/ChluIPFS.mock.js

```diff
@@ -0,0 +1,68 @@
+const constants = require('./constants');
+const time = require('./utils/timing');
+
+class ChluIPFS {
+
+    constructor(options = {}) {
+        if (Object.values(constants.types).indexOf(options.type) < 0) {
+            throw new Error('Invalid type');
+        }
+        this.type = options.type;
+    }
+    
+    async start(){
+        await time.milliseconds(1000);
```

Comment:
The time out thing is interesting. I can see it will be nice in the development mode. But it might unnecessarily slow us down for testing in a CI env. Can this be made optional? or maybe we have production, development, testing, three different modes? What do you think?

Reply:
I put these in to simulate that these actions take some time with a real implementation. They are already disabled in the tests because the time.milliseconds function is mocked to return immediately so there are no artificial slowdowns when running the automated tests

At the moment I think we don't need three separate modes